mne/epochs.py

```diff
@@ -809,14 +809,22 @@ def __next__(self, *args, **kwargs):
         """Provide a wrapper for Py3k."""
         return self.next(*args, **kwargs)
 
-    def average(self, picks=None):
-        """Compute average of epochs.
+    def average(self, picks=None, method="mean"):
+        """Compute an aggregate over epochs. Defaults to the mean.
```

Comment:
`over epochs, by default a mean.` (The two periods is a little bit odd. If you want to keep it this way, let's make it a separate line below.)